### ISSUE TITLE
How to train binary descriptor in image retrieval use this project

### ISSUE THREAD

**Issue by ghoshaw** [KevinMusgrave/pytorch-metric-learning#493]
HI, could you give some example for how to use the loss in this project to train binary descriptor in image retrieval ? like in this paper 'Simultaneous Feature Learning and Hash Coding with Deep Neural Networks'.  Thanks!

**Comment by KevinMusgrave** [comment]
I don't think this library will work for that. Check out these libraries instead:

https://github.com/swuxyj/DeepHash-pytorch
https://github.com/thulab/DeepHash

**Comment by ghoshaw** [comment]
ok, thanks
